### ISSUE TITLE
export csv double/float columns with decimal notation

### ISSUE THREAD

**Issue by mklechan** [Rdatatable/data.table#2434]
exports to csv where:
```
a
1.3
1
1
1
1
```
For h2o-3 parsing inter-operability, we should have 1.0 not 1. 
Could we add decimal notation to all values in these columns.

**Comment by MichaelChirico** [comment]
Hmm. Seems like a way to blow up file sizes unnecessarily? Is my first
reaction...

**Comment by mklechan** [comment]
Yes we sacrifice the file size, in return for better parsing and type detection of numeric columns.  The parsing in h2o-3 scans the first few lines of a file to determine the type. If the notation is consistent throughout the column, it will lead to less errors.
